js/client/modules/@arangodb/process-utils.js

```diff
@@ -1554,7 +1555,18 @@ function checkClusterAlive(options, instanceInfo, addArgs) {
   instanceInfo.arangods.forEach(arangod => {
     // agents don't support the ID call...
     if ((arangod.role !== "agent") && (arangod.role !== "single")) {
-      const reply = download(arangod.url + '/_db/_system/_admin/server/id', '', makeAuthorizationHeaders(instanceInfo.authOpts));
+      let reply;
+      try {
+        reply = download(arangod.url + '/_db/_system/_admin/server/id', '', makeAuthorizationHeaders(instanceInfo.authOpts));
+      } catch (e) {
+        print(RED + Date() + " error requesting agent '" + JSON.stringify(arangod) + "' Error: " + JSON.stringify(e) + RESET);
```

Comment:
This error message seems wrong. The `if` condition before it makes sure the URL is *not* called on agents, but the error message says "error requesting agent...". Should rather be "error requesting server"
```suggestion
        print(RED + Date() + " error requesting server '" + JSON.stringify(arangod) + "' Error: " + JSON.stringify(e) + RESET);
```